frontend/containers/layouts/discover-search/component.tsx

```diff
@@ -41,185 +38,143 @@ export const DiscoverSearch: FC<DiscoverSearchProps> = ({
                   id: 'BZG0d+',
                 })}
+                onClick={handleClickInput}
               />
             </div>
-            <div
-              className={cx(
-                'flex items-center gap-4 sm:gap-6 sm:justify-self-end transition-all ease-in-out duration-300 flex-grow-0',
-                {
-                  'w-0 h-0 overflow-hidden opacity-0': openFilters,
-                  'w-auto opacity-100': !openFilters,
-                }
-              )}
-            >
-              {/* Filters accordion header https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html */}
-              <h3>
-                <Button
-                  className="inline px-4 py-2 font-normal text-center bg-white text-green-dark hover:text-green-light focus-visible:outline-green-dark"
-                  id="filters-button"
-                  theme="naked"
-                  onClick={() => setOpenFilters(!openFilters)}
-                  aria-expanded={openFilters}
-                  aria-controls="filters"
-                >
-                  <FormattedMessage defaultMessage="Filters" id="zSOvI0" />
-                  <span
-                    className={cx(
-                      'flex items-center justify-center h-6 px-0 py-0 rounded-full border border-bg-green-light bg-green-light bg-opacity-30 text-green-dark font-bold text-xs transition-all ease-in',
-                      {
-                        'opacity-100 w-6 ml-2': !!filtersQuantity,
-                        'opacity-0 w-0 ml-0': !filtersQuantity,
-                      }
-                    )}
-                  >
-                    {filtersQuantity}
-                  </span>
+            {showSuggestion && (
+              <div>
+                <Button theme="naked" size="smallest" icon={CloseIcon} onClick={clearInput} />
+              </div>
+            )}
+            {pathname === Paths.Home && (
+              <div className="hidden md:block">
+                <Button className="text-green-dark" theme="naked" to={Paths.Discover}>
+                  {formatMessage({ defaultMessage: 'See full catalogue', id: 'oG/A0q' })}
                 </Button>
-              </h3>
-              <Button type="submit" className="h-full font-normal" theme="primary-orange">
-                {searchInputValue ? (
-                  <FormattedMessage defaultMessage="Search" id="xmcVZ0" />
-                ) : (
-                  searchButtonText
-                )}
-              </Button>
-            </div>
+              </div>
+            )}
+            {!openFilters && !showSuggestion && (
+              <div className="flex items-center flex-grow-0 gap-4 overflow-hidden transition-all duration-300 ease-in-out sm:gap-6 sm:justify-self-end">
+                {/* Filters accordion header https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html */}
+                <h3>
+                  <Button
```

Comment:
The focus ring is barely visible:
![The focus ring of the button is cut on all sides](https://user-images.githubusercontent.com/6073968/190176569-cb454701-46f2-4149-a973-4443254bf21f.png)